scripts/globals/artisan.lua

```diff
@@ -0,0 +1,91 @@
+-----------------------------------
+--
+--  Artisan Moogles
+--
+-----------------------------------
+require('scripts/globals/zone')
+require('scripts/globals/status')
+-----------------------------------
+
+tpz = tpz or {}
+tpz.artisan = tpz.artisan or {}
+
+local event = {
+    [tpz.zone.BASTOK_MARKETS] = 544,
+    [tpz.zone.WINDURST_WOODS] = 833,
+    [tpz.zone.RULUDE_GARDENS] = 10162,
+    [tpz.zone.SOUTHERN_SAN_DORIA] = 960
+}
+local menuFlags = {
+	expand = 0x8,
+	abags = 0x4,
+	aexpand = 0x2,
+}
+
+local getEpochDays = function()
+    local time = os.date("*t")
+    return time.year*365 + time.yday
+end
```

Comment:
Might wanna use `getMidnight()` instead, will keep this in line with everything else that resets at "JP" midnight.

Reply:
I wasn't aware that function existed. I will get that changed.

Reply:
Updated.